src/doc/src/reference/specifying-dependencies.md

```diff
@@ -162,19 +162,19 @@ is ignored and should not be used in version requirements.
 > [#10599]).
 >
 > Avoid constraining the upper bound of a version to be anything less than the
-> next semver incompatible version
-> (e.g. avoid `">=2.0, <2.4"`) as other packages in the dependency tree may
+> next semver incompatible version (e.g. avoid `">=2.0, <2.4"` or `"2.0.*"`),
```

Comment:
What about `~2.1.0`, isn't it effectively the same?

Reply:
Added that too. Thanks!